meta/sai_meta_hostintf.cpp

```diff
@@ -92,6 +92,117 @@ const char* metadata_sai_hostif_type_t_enum_values_short_names[] = {
+    SAI_HOSTIF_TRAP_ID_ARP_REQUEST,
+    SAI_HOSTIF_TRAP_ID_ARP_RESPONSE,
+    SAI_HOSTIF_TRAP_ID_DHCP,
+    SAI_HOSTIF_TRAP_ID_OSPF,
+    SAI_HOSTIF_TRAP_ID_PIM,
+    SAI_HOSTIF_TRAP_ID_VRRP,
+    SAI_HOSTIF_TRAP_ID_BGP,
+    SAI_HOSTIF_TRAP_ID_DHCPV6,
+    SAI_HOSTIF_TRAP_ID_OSPFV6,
+    SAI_HOSTIF_TRAP_ID_VRRPV6,
+    SAI_HOSTIF_TRAP_ID_BGPV6,
+    SAI_HOSTIF_TRAP_ID_IPV6_NEIGHBOR_DISCOVERY,
+    SAI_HOSTIF_TRAP_ID_IPV6_MLD_V1_V2,
+    SAI_HOSTIF_TRAP_ID_IPV6_MLD_V1_REPORT,
+    SAI_HOSTIF_TRAP_ID_IPV6_MLD_V1_DONE,
+    SAI_HOSTIF_TRAP_ID_MLD_V2_REPORT,
+    SAI_HOSTIF_TRAP_ID_IP2ME,
+    SAI_HOSTIF_TRAP_ID_SSH,
+    SAI_HOSTIF_TRAP_ID_SNMP,
+    SAI_HOSTIF_TRAP_ID_L3_MTU_ERROR,
+    SAI_HOSTIF_TRAP_ID_TTL_ERROR,
+    SAI_HOSTIF_TRAP_ID_CUSTOM_EXCEPTION_RANGE_BASE,
+};
+const char* metadata_sai_hostif_trap_type_t_enum_values_names[] = {
+    "SAI_HOSTIF_TRAP_TYPE_STP",
+    "SAI_HOSTIF_TRAP_TYPE_LACP",
+    "SAI_HOSTIF_TRAP_TYPE_EAPOL",
+    "SAI_HOSTIF_TRAP_TYPE_LLDP",
+    "SAI_HOSTIF_TRAP_TYPE_PVRST",
+    "SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_QUERY",
+    "SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_LEAVE",
+    "SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V1_REPORT",
+    "SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V2_REPORT",
+    "SAI_HOSTIF_TRAP_TYPE_IGMP_TYPE_V3_REPORT",
+    "SAI_HOSTIF_TRAP_TYPE_SAMPLEPACKET",
+    "SAI_HOSTIF_TRAP_TYPE_ARP_REQUEST",
+    "SAI_HOSTIF_TRAP_TYPE_ARP_RESPONSE",
+    "SAI_HOSTIF_TRAP_TYPE_DHCP",
+    "SAI_HOSTIF_TRAP_TYPE_OSPF",
+    "SAI_HOSTIF_TRAP_TYPE_PIM",
+    "SAI_HOSTIF_TRAP_TYPE_VRRP",
+    "SAI_HOSTIF_TRAP_TYPE_BGP",
+    "SAI_HOSTIF_TRAP_TYPE_DHCPV6",
+    "SAI_HOSTIF_TRAP_TYPE_OSPFV6",
+    "SAI_HOSTIF_TRAP_TYPE_VRRPV6",
+    "SAI_HOSTIF_TRAP_TYPE_BGPV6",
+    "SAI_HOSTIF_TRAP_TYPE_IPV6_NEIGHBOR_DISCOVERY",
+    "SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_V2",
+    "SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_REPORT",
+    "SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_DONE",
+    "SAI_HOSTIF_TRAP_TYPE_MLD_V2_REPORT",
+    "SAI_HOSTIF_TRAP_TYPE_IP2ME",
+    "SAI_HOSTIF_TRAP_TYPE_SSH",
+    "SAI_HOSTIF_TRAP_TYPE_SNMP",
+    "SAI_HOSTIF_TRAP_TYPE_L3_MTU_ERROR",
+    "SAI_HOSTIF_TRAP_TYPE_TTL_ERROR",
+    "SAI_HOSTIF_TRAP_TYPE_CUSTOM_EXCEPTION_RANGE_BASE",
+    NULL
+};
+const char* metadata_sai_hostif_trap_type_t_enum_values_short_names[] = {
+    "STP",
+    "LACP",
+    "EAPOL",
+    "LLDP",
+    "PVRST",
+    "IGMP_TYPE_QUERY",
+    "IGMP_TYPE_LEAVE",
+    "IGMP_TYPE_V1_REPORT",
+    "IGMP_TYPE_V2_REPORT",
+    "IGMP_TYPE_V3_REPORT",
+    "SAMPLEPACKET",
+    "ARP_REQUEST",
+    "ARP_RESPONSE",
+    "DHCP",
+    "OSPF",
+    "PIM",
+    "VRRP",
+    "BGP",
+    "DHCPV6",
+    "OSPFV6",
+    "VRRPV6",
+    "BGPV6",
+    "IPV6_NEIGHBOR_DISCOVERY",
+    "IPV6_MLD_V1_V2",
+    "IPV6_MLD_V1_REPORT",
+    "IPV6_MLD_V1_DONE",
+    "MLD_V2_REPORT",
+    "IP2ME",
+    "SSH",
+    "SNMP",
+    "L3_MTU_ERROR",
+    "TTL_ERROR",
+    "CUSTOM_EXCEPTION_RANGE_BASE",
+    NULL
+};
+const size_t metadata_sai_hostif_trap_type_t_enum_values_count = 33;
```

Comment:
this is temprorary its copy from auto generated metadata and it needs to be this way since value needs to be known on compile time